upstream-sha256: c46ab546c87e7857c7e86458aff9474223af8fc7a3c0fbb1987212c319efcba6
actual-sha256: ae567f5e38e8c7cb4270e9c9783e2397ebbfb318332dc061ab8ab6002145d8ef
--- upstream.expected
+++ tolk.actual
@@ -16,9 +16,9 @@
     float4 val1 = (*((float4*)((data1_1769472+(alu4+768)))));
     float4 val2 = (*((float4*)((data1_1769472+alu4))));
     float4 val3 = (*((float4*)((data2_768+alu3))));
-    *(buf0+0) = ((*(buf0+0))+(val2.x*val3.x)+(val2.y*val3.y)+(val2.z*val3.z)+(val2.w*val3.w));
-    *(buf0+1) = ((*(buf0+1))+(val1.x*val3.x)+(val1.y*val3.y)+(val1.z*val3.z)+(val1.w*val3.w));
-    *(buf0+2) = ((*(buf0+2))+(val0.x*val3.x)+(val0.y*val3.y)+(val0.z*val3.z)+(val0.w*val3.w));
+    *(buf0+0) = ((*(buf0+0))+((val2.x*val3.x)+(val2.y*val3.y)+(val2.z*val3.z)+(val2.w*val3.w)));
+    *(buf0+1) = ((*(buf0+1))+((val1.x*val3.x)+(val1.y*val3.y)+(val1.z*val3.z)+(val1.w*val3.w)));
+    *(buf0+2) = ((*(buf0+2))+((val0.x*val3.x)+(val0.y*val3.y)+(val0.z*val3.z)+(val0.w*val3.w)));
   }
   int alu9 = ((gidx0*96)+(lidx0*3));
   int alu10 = (alu9+1);
